src/resources/views/fields/latlng.blade.php

```diff
@@ -0,0 +1,204 @@
+                @else
+                    {{ $attribute }}="{{ $value }}"
+                @endif
+            @endif
+        @endforeach
+        >
+    </div>
+    <div class="form-group col-md-12">
+        <div id="map-canvas" style="{{$field['map_style']}}"></div>  
+    </div>
+
+{{-- ########################################## --}}
+{{-- Extra CSS and JS for this particular field --}}
+{{-- If a field type is shown multiple times on a form, the CSS and JS will only be loaded once --}}
+@if ($crud->checkIfFieldIsFirstOfItsType($field, $fields))
+
+    {{-- FIELD CSS - will be loaded in the after_styles section --}}
+    @push('crud_fields_styles')
+	    {{-- YOUR CSS HERE --}}
+    @endpush
+
+    {{-- FIELD JS - will be loaded in the after_scripts section --}}
+    @push('crud_fields_scripts')
+    	{{-- YOUR JS HERE --}}
+        <script src="https://maps.googleapis.com/maps/api/js?v=3&libraries=places&key={{$field['google_api_key']}}"></script>
+
+        <script>
+        $(document).ready(function() {
+
+            // overwrite BackPack-CRUD submit on Enter
+            $(window).keydown(function(event){
+                if(event.keyCode == 13) {
+                    event.preventDefault();
+                    return false;
+                }
+            });
+
+            function initialize() {
+                var marker;
+                var default_zoom = {{$field['default_zoom']}};
+                var geolocate_icon = "{{$field['geolocate_icon']}}";
+                var latlong = document.getElementById('latlong').value; // get latlng value if any
+                latlong = latlong.split(',');
+                var latlng = (latlong == '') ? new google.maps.LatLng(3.138675,101.6167769) : new google.maps.LatLng(latlong[0], latlong[1]);
+                var geocoder = new google.maps.Geocoder();
+                var mapOptions = {
+                    center: latlng,
+                    zoom: 8,
+                    mapTypeId: google.maps.MapTypeId.ROADMAP,
+                    draggableCursor: "pointer",
+                    streetViewControl: false
+                };
+
+                // set search_address with formatted_address if latlng has value
+                (latlong == '') ? '' : setGeoCoder(latlng);
+
+                // set marker icon
+                var marker_icon = "{{$field['marker_icon']}}";
+                (marker_icon == '') ? null : marker_icon;
+                
+                var map = new google.maps.Map(document.getElementById('map-canvas'), mapOptions);
+            
+                var markers = new google.maps.Marker({
+                    position: latlng,
+                    map: map,
+                    icon : marker_icon
+                });
+            
+                google.maps.event.addListener(map, "click", function (location) {
+                    //map.setCenter(location.latLng); // set location to map's center each time map is clicked
```

Comment:
Why's that commented out?

Reply:
I believe I leave that as an option. To tell user that the option exists if they want to include it.